src/pages/RealtimeMapPage.tsx

```diff
@@ -46,8 +46,8 @@ export function numberToColorHsl(i: number, max: number) {
   return `hsl(${hue}, ${saturation}%, ${lightness}%)`
 }
 
-const fiveMinutesAgo = moment().subtract(5, 'minutes')
-const fourMinutesAgo = fiveMinutesAgo.add(1, 'minutes')
+const now = moment()
+const oneMinuteAgo = moment().subtract(1, 'minutes')
```

Comment:
We should now use `now`, because the API can't guarantee this information will be available. The ETL process may take some time

Reply:
> We should now use `now`, because the API can't guarantee this information will be available. The ETL process may take some time

Can we do that?:
```
const now = new Date()
const oneMinuteAgo = moment().subtract(1, 'minutes')
const [from, setFrom] = useState(oneMinuteAgo)
const [to, setTo] = useState(moment(now))
```

Reply:
My bad - I wanted to say we should not use `now`.
If you'll try to load data that hasn't been processed (yet), the backend will return an empty array and the frontend code will memorize it

Reply:
> My bad - I wanted to say we should not use `now`. If you'll try to load data that hasn't been processed (yet), the backend will return an empty array and the frontend code will memorize it

I'm trying to understand the issue. I read it the first time "not" actually:) We should not use now: 
I thought you meant the moment api, but now I understand that now as a variable is problematic. 

So, as a default, we want the application first render to be four minutes from now (=to in the state), and five minutes ago (= from).

Reply:
exactly